concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1098	1098	hinauf-, hinabsteigen	hinabsteigen	to descend	2014	GO DOWN (DESCEND)
+Steinthal-1875-1565-1099	1099	ausruhen	ausruhen	to rest	168	REST
+Steinthal-1875-1565-1100	1100	zurückbleiben	zurückbleiben	to stay behind		
+Steinthal-1875-1565-1101	1101	entgegengehn	entgegengehn	to approach	1778	APPROACH
+Steinthal-1875-1565-1102	1102	begegnen	begegnen	to meet	1831	MEET
+Steinthal-1875-1565-1103	1103	sich trennen	sich trennen	to split up		
+Steinthal-1875-1565-1104	1104	grüssen	grüssen	to greet	3552	GREET
+Steinthal-1875-1565-1105	1105	Ceremonien beim Grüssen (etwa: Hand geben)	Ceremonien beim Grüssen	ceremonies of greeting		
+Steinthal-1875-1565-1106	1106	Ceremonien beim Grüssen (etwa: Hand geben)	Hand geben	to shake hands		
+Steinthal-1875-1565-1107	1107	Abschied nehmen	Abschied nehmen	to say farewell		
+Steinthal-1875-1565-1108	1108	sich auf der Reise irren	sich auf der Reise irren	to lose one‘s way		
+Steinthal-1875-1565-1109	1109	den Weg zeigen (überhaupt etwas zeigen)	den Weg zeigen	to show the way		
+Steinthal-1875-1565-1110	1110	den Weg zeigen (überhaupt etwas zeigen)	überhaupt etwas zeigen	to show something	1039	SHOW
+Steinthal-1875-1565-1111	1111	führen	führen	to lead	43	LEAD (GUIDE)
+Steinthal-1875-1565-1112	1112	schicken	schicken	to send	1127	SEND
+Steinthal-1875-1565-1113	1113	Bote	Bote	messenger		
+Steinthal-1875-1565-1114	1114	Er hat mich gut aufgenommen.	Er hat mich gut aufgenommen.	He accommodated me well.		
+Steinthal-1875-1565-1115	1115	an einem Orte verweilen	an einem Orte verweilen	to stay in one place		
+Steinthal-1875-1565-1116	1116	Du bist lange geblieben.	Du bist lange geblieben.	You have stayed long.		
+Steinthal-1875-1565-1117	1117	schneiden	schneiden	to cut	1432	CUT
+Steinthal-1875-1565-1118	1118	Ein Stock bricht.	Ein Stock bricht.	A stick breaks.		
+Steinthal-1875-1565-1119	1119	den Stock zerbrechen	den Stock zerbrechen	to break a stick		
+Steinthal-1875-1565-1120	1120	Er ist zerbrochen.	Er ist zerbrochen.	It is broken.	588	BROKEN
+Steinthal-1875-1565-1121	1121	etwas zerreissen	etwas zerreissen	to tear apart		
+Steinthal-1875-1565-1122	1122	Es ist zerrissen.	Es ist zerrissen.	It is torn.		
+Steinthal-1875-1565-1123	1123	Das Kleid ist abgetragen.	Das Kleid ist abgetragen.	The dress is worn out.		
+Steinthal-1875-1565-1124	1124	Der Krug ist gespalten.	Der Krug ist gespalten.	The jug is split in two.		
+Steinthal-1875-1565-1125	1125	Der Schlauch platzt.	Der Schlauch platzt.	The pipe bursts.		
+Steinthal-1875-1565-1126	1126	werfen	werfen	to throw	1456	THROW
+Steinthal-1875-1565-1127	1127	setzen	setzen	to sit down	1649	SIT DOWN
+Steinthal-1875-1565-1128	1128	legen	legen	to lay	2071	LAY (VERB)
+Steinthal-1875-1565-1129	1129	sitzen	sitzen	to sit	1416	SIT
+Steinthal-1875-1565-1130	1130	liegen	liegen	to lie	1411	LIE (REST)
+Steinthal-1875-1565-1131	1131	binden	binden	to bind	1917	TIE
+Steinthal-1875-1565-1132	1132	knüpfen	knüpfen	to tie	1917	TIE
```

Comment:
Double mapping

Reply:
If knüpfen here is associated with the craft of making carpets by hand (as opposed to 'weaving a carpet', i.e. 'knotting a carpet' (https://en.wikipedia.org/wiki/Knotted-pile_carpet)) I'd unmap this. We only have the noun KNOT so far.